AirlineReservation/Models/Data/AirlineModel.cs

```diff
@@ -0,0 +1,21 @@
+﻿using MongoDB.Bson;
+using MongoDB.Bson.Serialization.Attributes;
+
+namespace Airline_Reservation.Models.Data
```

Comment:
namsespaces are usually in Pascal case
Instead of using nested namespace brackets, we can utilize the using namespace directive